pkg/util/metricsinfo/quota_metric.go

```diff
@@ -84,7 +84,7 @@ type QueryNodeQuotaMetrics struct {
 }
 
 type DataCoordQuotaMetrics struct {
-	TotalBinlogSize int64
+	CollectionBinlogSize map[int64]int64
 }
```

Comment:
RootCoord upgrade may cause compatability issue here. Since older datacoord does not has `CollectionBinlogsSize` field, the quota limit may be broken here